src/pages/TagCreator/Organization.js

```diff
@@ -134,15 +135,15 @@ export const OrgChange = ({ orgName, setOrgTags, changeValue, setDisplayState })
   const handleSubmitOrg = () => {
     const topics = []
     if (orgName) {
-      axios.get(`${process.env.REACT_APP_API_URL}/api/organizations/` + orgName,)
+      const og = orgName.replace(/ /g,"-").toLowerCase()
+      axios.get(`${process.env.REACT_APP_API_URL}/api/organizations/`+og,)
         .then(res => {
-          let po = res.data.parent_organization
+          const po = res.data.parents
           if (res.data.org_tag !== "") {
             topics.push(res.data.org_tag)
           }
-          while (po != null) {
-            topics.push(po.org_tag)
-            po = po.parent_organization
+          if (po.length!==0){
+            po.map(p=>(p.org_tag !== "") ? topics.push(p.org_tag) : null)
```

Comment:
Can spacing be update a bit?  Otherwise looks good

Reply:
adjusted spacing,  it expects a return statement.
         if (po.length !== 0){
            po.map(p =>{
              if (p.org_tag !== ""){
                topics.push(p.org_tag)
              }
            })
          }